src/components/form/new.vue

```diff
@@ -357,7 +357,6 @@ export default {
       "chooseOne": "vyberte jeden"
     },
     "action": {
-      "upload": "Nahrát",
```

Comment:
The source message is "Upload", which was moved to en.json5. I used `@transifexKey`, so this message should appear in cs.json without retranslation.